defaults/main.yml

```diff
@@ -2,8 +2,12 @@
 node_exporter_version: 0.16.0
 node_exporter_web_listen_address: "0.0.0.0:9100"
 
+node_exporter_textfile_dir: "/opt/node_exporter/textfile"
```

Comment:
Shouldn't it be somewhere in `/var`? Maybe in `/var/local`?

Reply:
I'd say '/var/lib`, how about that?

Reply:
LGTM :+1: